src/Slider.jsx

```diff
@@ -4,7 +4,7 @@ import linear from './algorithms/linear';
 
 // istanbul ignore next
 function getDOMNode(node) {
-  return React.version.indexOf('0.14') > -1 ? node : node.getDOMNode();
+  return React.version.replace('.', '') >= 140 ? node : node.getDOMNode();
```

Comment:
I'm wondering if this is actually the best way to check if it's 0.14 or 15.0 or newer :-/